rhobot.go

```diff
@@ -14,66 +15,111 @@ func main() {
 
 	app := cli.NewApp()
 	app.Name = "Rhobot"
-	app.Usage = "Rhobot is your friend."
+	app.Usage = "Rhobot is a database development tool that uses DevOps best practices."
 	app.EnableBashCompletion = true
 
+	config := config.NewConfig()
+
 	app.Commands = []cli.Command{
 		{
 			Name:    "run",
 			Aliases: []string{},
-			Usage:   "healthchecks|gocd|tbd",
+			Usage:   "healthchecks|pipeline|tbd",
 			Subcommands: []cli.Command{
 				{
 					Name:  "healthchecks",
-					Usage: "[database uri] [path to healthcheck file]",
+					Usage: "HEALTHCHECK_FILE [DATABASE_URI]",
 					Action: func(c *cli.Context) {
-						dburi := c.Args()[0]
-						path := c.Args()[1]
-						fmt.Println("DB_URI: ", dburi)
+
```

Comment:
consider using: func (a Args) Get(n int) string 
instead of checking the length of the Args array.
If DATABASE_URI is missing, the Get function will return ""